spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -35,6 +35,15 @@
       expect(flash[:success]).to eq(I18n.t("blacklight.delete.success", count: 3))
       media_objects.each {|mo| expect(MediaObject.exists?(mo.id)).to be_falsey }
     end
+    it "should remove more than the blacklight default number of items (>10)" do
+      8.times do
```

Comment:
Why does 8 times give you 11 total deletions?  Could you put a comment to explain this?